builder/googlecompute/config.go

```diff
@@ -73,7 +73,13 @@ func NewConfig(raws ...interface{}) (*Config, []string, error) {
 	}
 
 	if c.ImageName == "" {
-		c.ImageName = "packer-{{timestamp}}"
+		img, err := interpolate.Render("packer-{{timestamp}}", nil)
+		if err != nil {
+			panic(err)
```

Comment:
I changed this to use multi-error. However, this should only happen if the interpolate implementation is broken so panic might be legit in this case.